llvm/test/CodeGen/AArch64/GlobalISel/split-offsets-for-stp.ll

```diff
@@ -1,6 +1,6 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 3
-; RUN: llc -mtriple=aarch64-apple-ios -verify-machineinstrs -global-isel -aarch64-postlegalizer-consecutive-memops=0 < %s | FileCheck %s --check-prefix=CHECK-NO-SPLIT
-; RUN: llc -mtriple=aarch64-apple-ios -verify-machineinstrs -global-isel < %s | FileCheck %s --check-prefix=CHECK-SPLIT
+; RUN: llc -mtriple=aarch64-apple-ios -verify-machineinstrs -global-isel -aarch64-enable-sink-fold=true -aarch64-postlegalizer-consecutive-memops=0 < %s | FileCheck %s --check-prefix=CHECK-NO-SPLIT
```

Comment:
Why are you adding this flag here if the opt isn't enabled by default? The test output is unchanged.

Reply:
I believe the idea is that this will be enabled by default, and this patch fixes the regressions that would otherwise be cause by turning it on.

Reply:
Yes, that's right, for several other tests in the testsuite as well.

Reply:
I see, LGTM.